test/conformance/tests/sriov_operator.go

```diff
@@ -681,292 +423,566 @@ var _ = Describe("[sriov] operator", func() {
 				})
 			})
 		})
-		Context("Resource Injector", func() {
-			// 25815
-			It("Should inject downward api volume", func() {
-				node := sriovInfos.Nodes[0]
-				intf, err := sriovInfos.FindOneSriovDevice(node)
-				Expect(err).ToNot(HaveOccurred())
-
-				nodePolicy := &sriovv1.SriovNetworkNodePolicy{
-					ObjectMeta: metav1.ObjectMeta{
-						GenerateName: "test-apivolumepolicy",
-						Namespace:    operatorNamespace,
-					},
+	})
 
-					Spec: sriovv1.SriovNetworkNodePolicySpec{
-						NodeSelector: map[string]string{
-							"kubernetes.io/hostname": node,
-						},
-						NumVfs:       5,
-						ResourceName: "apivolresource",
-						Priority:     99,
-						NicSelector: sriovv1.SriovNetworkNicSelector{
-							PfNames: []string{intf.Name},
-						},
-						DeviceType: "netdevice",
-					},
-				}
+	Describe("Requiring new SriovNetworkNodePolicy", func() {
```

Comment:
Can this be "Custom SriovNetworkNodePolicy" ?

Reply:
Done